cypress-tests/cypress/e2e/PaymentUtils/Adyen.js

```diff
@@ -379,6 +396,78 @@ export const connectorDetails = {
         },
       },
     },
+    SaveCardUseNo3DSAutoCaptureOffSession: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
```

Comment:
Same here

Reply:
This is confirm call removing this will cause unintended issues